components/Card/card.css

```diff
@@ -0,0 +1,110 @@
+/* All Cards */
+
+.card, .card-badge{
+	border-radius: 0.5rem;
+	font-family: var(--font-secondary);
+}
+.card{
+	position:relative;
+	max-width: 19rem;
+	border: 1px solid var(--clr-gray);
+}
+
+.card .card-img-top {
+	width: 100%;
+	object-fit: cover;
+	object-position: top;
+	height: 40vh;
+	border-top-left-radius: 0.5rem;
+	border-top-right-radius: 0.5rem;
+}
+
+.card-body{
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+	padding: 0.5rem;
+}
+
+.card-title{
+	font-family: var(--font-primary);
+}
+
+.card-desc{
+	font-size: 1.1rem;
+}
+
+.card-badge, .card-dismiss{
+	position: absolute;
+	font-size: var(--fs-sm);
+	color: var(--clr-gray);
+}
+
+/* Horizontal Card */
+
+.card-horizontal {
+	display: flex;
+	align-items: center;
+	height: fit-content;
+	max-width: 100%;
+	height: auto;
+	margin: 0 1rem;
+}
+
+.card-horizontal > .card-img-top{
+	border-radius: 0.5rem 0 0 0.5rem;
+	max-width: 50%;
+}
+
+/* Card with a Badge */
+
+.card-badge{
+	background: var(--clr-white);
+	font-weight: bold;
+	padding: 0.1rem 0.5rem;
+	border-radius: 0.5rem;
+	top: 0.5rem;
+	left: 0;
+}
+
+/* Card with dismiss */
+.card-dismiss{
+	top: 0.3rem;
+	right: 1rem;
+}
+
+/* Card with text-overlay */
+.card-text-overlay{
+	position: absolute;
+	border-radius: 0.5rem;
+	background-color: var(--clr-gray);
+	color: var(--clr-white);	
+	top: 0;
+	bottom:0;
+	right: 0;
+	left:0;
+	opacity: 0.8;
+	z-index: 2;
+	font-family: var(--font-primary);
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+/* Card with Shadow */
+.card-shadow{
+	/* box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px; */
+	box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+}
```

Comment:
- You can remove this comment to make your code cleaner.
- You can use CSS variables for box-shadow values
- Apart from the points mentioned by @manojasarsa , everything looks good🙌